frontend/templates/views/partials/version-toggle.j2

```diff
@@ -1,10 +1,12 @@
-{# Warning: This is a special template rendered through Nunjucks in platform/lib/pipeline/markdownDocument.js #}
-{# Since it doesn't use grow, global page context is not available. #}
 <h1>
-{{title}}
+{{doc.title}}
+[% if versions and versions.length > 1 %]
 <div class="ap--versions">
-  {% for v in versions %}
-    <a{% if v !== version %} href="../{{title}}-v{{v}}.html"{% endif %}>{{v}}</a>
+  {% for v in doc.versions %}
+  [% if versions and versions.includes('{{v}}') %]
+  <a{% if v != doc.version %} href="../{{doc.title}}-v{{v}}.html?format=[= format =]"{% endif %}>{{v}}</a>
```

Comment:
Try to use doc.slug to avoid the problems with components where the title is not exactly the component name like in the translations of amp-ad.

Reply:
done